### PR TITLE
New full-stack demo

### PR DESCRIPTION
**Pull request by henri-tremblay**
@anthonydahanne Can you have a look and make sure everything is working fine? We might really want to have fake calls to google and darksky to make it easier to use.

One thing that I don't like is that we can't pass a variable for the terracotta cluster. But I think we now have system properties replacement. So we should use this probably. Or find a more "Springy" way to configure the CacheProvider

### CODE REVIEW

**Comment by anthonydahanne** [comment]
@henri-tremblay I haven't reviewed yet but I was wondering the reason why all the commits have changed since the Innovation week ?

**Comment by anthonydahanne** [comment]
```
./mvnw clean install
```

eventually failed with : 

```

Results :

Tests in error: 
  ActorResourceIntTest.createActor » OutOfMemory Java heap space
  ActorResourceIntTest.deleteActor » OutOfMemory Java heap space
  ActorResourceIntTest.updateActor » OutOfMemory Java heap space

```

Tests should not load the actors list from the internet; a (small) list should be provided

File: fullstack/README.md [hunk 1]
```diff
@@ -1,23 +1,51 @@
...
+* Hibernate 5 reusing the same `CacheManager` as Spring through JCache  
+  * [application-dev.xml](src/main/resources/config/application-dev.yml), [application-prod.xml](src/main/resources/config/application-prod.yml): Hibernate cache provider configuration
+  * [SpringCacheRegionFactory.java](src/main/java/org/terracotta/demo/config/jcache/SpringCacheRegionFactory.java): Hibernate region factory linked with Spring
+* Push JCache JMX metrics to [Dropwizard Metrics](http://metrics.dropwizard.io)
+  * [MetricsConfiguration.java](src/main/java/org/terracotta/demo/config/MetricsConfiguration.java)
+  * [JCacheGaugeSet.java](src/main/java/org/terracotta/demo/config/JCacheGaugeSet.java)
+* Ehcache clustered configuration for production
+  * [ehcache-clustered.xml](src/main/resources/ehcache-clustered.xml): Ehcache clustered configuration
+  * [application-prod.xml](src/main/resources/config/application-prod.yml): Clustered configuration in Spring 
+* Teracotta server deployed with Docker
```
Comment (anthonydahanne) [inline fullstack/README.md]: typo : Terracotta

File: fullstack/pom.xml [hunk 2]
```diff
@@ -6,119 +6,70 @@
...
     <properties>
-        <hikaricp.version>2.4.6</hikaricp.version>
         <assertj-core.version>3.5.2</assertj-core.version>
         <awaitility.version>1.7.0</awaitility.version>
         <commons-io.version>2.5</commons-io.version>
         <commons-lang.version>3.4</commons-lang.version>
         <frontend-maven-plugin.version>1.0</frontend-maven-plugin.version>
-        <gatling.version>2.1.7</gatling.version>
         <gatling-maven-plugin.version>2.1.7</gatling-maven-plugin.version>
+        <gatling.version>2.1.7</gatling.version>
```
Comment (anthonydahanne) [inline fullstack/pom.xml]: why reshuffling so many lines in this file ?
history is not readable anymore...

File: README.md [hunk 3]
```diff
@@ -2,11 +2,12 @@
 
 This repository contains examples of and tutorials about usage of Ehcache 3.
```
Comment (anthonydahanne) [inline README.md]: please provide original commit history

**Comment by henri-tremblay** [comment]
Yes. Because we are on a new gît repo. So I had to restart from scratch

Le 25 oct. 2016 4:09 PM, "Anthony Dahanne" <notifications@github.com> a
écrit :

**Comment by anthonydahanne** [comment]
please have a look at https://github.com/ehcache/ehcache3-samples/pull/7 it keeps history, remove the generated bower files and imports this current PR commits

**Comment by henri-tremblay** [comment]
Done by #7 instead

**Comment by henri-tremblay** [comment]
Bower should not be removed. The problem is that bower packages are based
on git tags. They are considered unstable. So committing them is deemed
safer.

Le 26 oct. 2016 6:55 AM, "Anthony Dahanne" <notifications@github.com> a
écrit :

**Pull request closed without merging**